src/PowerShellEditorServices/Debugging/StackFrameDetails.cs

```diff
@@ -86,16 +111,34 @@ public class StackFrameDetails
         static internal StackFrameDetails Create(
             PSObject callStackFrameObject,
             VariableContainerDetails autoVariables,
-            VariableContainerDetails localVariables)
+            VariableContainerDetails localVariables,
+            string workspaceRootPath = null)
         {
+            string moduleId = string.Empty;
+            var presentationHint = StackFramePresentationHint.Normal;
+
+            var invocationInfo = callStackFrameObject.Properties["InvocationInfo"].Value as InvocationInfo;
```

Comment:
Have you tested this with remote debugging?  Sometimes these objects come back as a weird serialized type that will cause issues.